.gitlab-ci.yml

```
@@ -89,11 +93,11 @@ before_script:
 
     - dev/ci/gitlab-section.sh start coq.build coq.build
     - make dunestrap
-    - dune build -p coq-core,coq-stdlib,coqide-server,coqide,coq
+    - dune build -p $PKGS
     - dev/ci/gitlab-section.sh end coq.build
 
     - dev/ci/gitlab-section.sh start coq.install coq.install
-    - dune install --prefix="$(pwd)/_install_ci" coq-core coq-stdlib coqide-server coqide coq
+    - dune install --prefix="$(pwd)/_install_ci" $(sed -e 's/,/ /g' <<< ${PKGS})
```

This idiom isn't that hard if you know, but it is gratuitous obfuscation:
```suggestion
    - dune install --prefix="$(pwd)/_install_ci" $(echo ${PKGS} | sed -e 's/,/ /g')
```

---

(This shouldn't block merge, but is motivated by @ejgallego 's comment and will help some people read it).

---

If we are talking about more concise and possibly more readable alternatives, I suggest `${PKGS//,/ }`.